generic/tests/cfg/pxe_boot.cfg

```diff
@@ -3,6 +3,7 @@
     type = pxe_boot
     requires_root = yes
     images = pxe
+    image_boot_pxe = no
```

Comment:
Well, the issue only happened on ppc. But, if this test wants to boot from nic firstly, then a disk should not be assigned bootindex number to 0, isn't it? So, I think it is a bonus for other platform.

If I misunderstand it, please correct me.